Src/BootCamp.Chapter/TextTable.cs

```diff
@@ -1,40 +1,77 @@
-﻿namespace BootCamp.Chapter
+﻿using System;
+using System.Text;
+
+namespace BootCamp.Chapter
 {
     /// <summary>
     /// Part 1.
     /// </summary>
     public static class TextTable
     {
         /*
-
          Input: "Hello", 0
            +-----+
            |Hello|
            +-----+
-           
+        */
+
+        public static string Build(string message, int padding)
+        {
+            var sb = new StringBuilder();
+            sb.Append($"+-----+{Environment.NewLine}|{ message}|{Environment.NewLine}+-----+");
+            Console.WriteLine(sb.ToString());
+            return sb.ToString();
+        }
+
+        /*
          Input: $"Hello{Environment.NewLine}World!", 0
            +------+
            |Hello |
            |World!|
            +------+
-           
+         */
+
+        public static string Build1(string message, int padding)
```

Comment:
Merge with build1. Also I think we have tests, please apply the same names so that it gets tested